src/acl/external/session/ext_session_acl.cc

```diff
@@ -46,9 +46,7 @@
 #if HAVE_TDB_H
 #include <tdb.h>
 #endif
-#if HAVE_UNISTD_H
-#include <unistd.h>
-#endif
+#include <cunistd>
```

Comment:
These converted headers should be moved up top with the other c++11 headers.
Or maybe better a followup to make `sort-includes.pl` to enforce the squid guidelines header listing better.

Reply:
> These converted headers should be moved up top with the other c++11 headers.
>  Or maybe better a followup to make sort-includes.pl to enforce the squid guidelines header listing better.

FWIW, I strongly recommend the second alternative -- it would make both changes a lot easier to review correctly, both during scripts development and during official review.